### PR TITLE
 roe-2423 add params to check your answers before sending your application - who can we contact page

### PR DESCRIPTION
**Pull request by HatiDev**
### JIRA link
https://companieshouse.atlassian.net/browse/ROE-2423

### Change description
Add params url to Check your answers before sending your application - Who can we contact page?

### Work checklist

- [ ] Tests added where applicable
- [ ] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: src/controllers/check.your.answers.controller.ts [hunk 1]
```diff
@@ -56,7 +56,7 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
 
 export const post = async (req: Request, res: Response, next: NextFunction) => {
   try {
-    logger.debugRequest(req, `POST ${config.CHECK_YOUR_ANSWERS_PAGE}`);
+    logger.debugRequest(req, `POST ${config.CHECK_YOUR_ANSWERS_WITH_PARAMS_URL}`);
```
Comment (markpit) [inline src/controllers/check.your.answers.controller.ts:59]: the controller is used by the non-params url as well so it's probably best to leave it as config.CHECK_YOUR_ANSWERS_PAGE

File: src/utils/navigation.ts [hunk 2]
```diff
@@ -208,22 +209,22 @@ export const NAVIGATION: Navigation = {
   [config.BENEFICIAL_OWNER_TYPE_URL]: {
     currentPage: config.BENEFICIAL_OWNER_TYPE_PAGE,
     previousPage: () => config.BENEFICIAL_OWNER_STATEMENTS_URL,
-    nextPage: [config.CHECK_YOUR_ANSWERS_URL, config.TRUST_INFO_URL]
+    nextPage: [config.CHECK_YOUR_ANSWERS_WITH_PARAMS_URL, config.TRUST_INFO_URL]
```
Comment (markpit) [inline src/utils/navigation.ts:212]: i think we only need to change the next page to a 'wiyth params' version if the config is for a 'with_params' page, this one is BENEFICIAL_OWNER_TYPE_URL which is the old type so we need that to go to next page without params, so leave as config.CHECK_YOUR_ANSWERS_URL.
same applies for the other changes, only change the next page if the config is for a 'with_params' page

**Comment by mwestacott** [comment]
Just checking, were any changes required to deliver AC2 of the JIRA ticket, or is that following in a later PR?

File: src/controllers/check.your.answers.controller.ts (hunk 1 above)
Comment (HatiDev) [reply src/controllers/check.your.answers.controller.ts]: I have edited the controller back to config.CHECK_YOUR_ANSWERS_PAGE

File: src/utils/navigation.ts (hunk 2 above)
Comment (HatiDev) [reply src/utils/navigation.ts]: edited according to above.

**Comment by HatiDev** [comment]
> Just checking, were any changes required to deliver AC2 of the JIRA ticket, or is that following in a later PR?

looks like it needs a separate story as it is on the presenter page.

File: src/utils/navigation.ts [hunk 3]
```diff
@@ -208,22 +208,22 @@ export const NAVIGATION: Navigation = {
   [config.BENEFICIAL_OWNER_TYPE_URL]: {
     currentPage: config.BENEFICIAL_OWNER_TYPE_PAGE,
     previousPage: () => config.BENEFICIAL_OWNER_STATEMENTS_URL,
-    nextPage: [config.CHECK_YOUR_ANSWERS_URL, config.TRUST_INFO_URL]
+    nextPage: [config.CHECK_YOUR_ANSWERS_PAGE, config.TRUST_INFO_URL]
```
Comment (dlloyd1-ch) [inline src/utils/navigation.ts:211]: What's the reason why this has changed from URL to PAGE? Looks like a potential breaking change. Have you run it locally? Same with line 221.

**Review by dlloyd1-ch — COMMENTED** [review]
There's only one file in this PR (navigation). Is that correct? Does that satisfy AC1 and AC3?

File: src/utils/navigation.ts (hunk 3 above)
Comment (markpit) [reply src/utils/navigation.ts:211]: yes, they should be URL as the PAGE is just a template name.

Comment (HatiDev) [reply src/utils/navigation.ts:211]: edited back to url. As for AC3 it may have to be a separate story as it relates to the same change links as on AC2

Comment (markpit) [reply src/utils/navigation.ts:211]: the change still shows as config.CHECK_YOUR_ANSWERS_PAGE, did you push up your changes as github says last commit was 4 days ago

Comment (HatiDev) [reply src/utils/navigation.ts]: pushed up now

**Comment by dlloyd1-ch** [comment]
@HatiDev Can you close this PR if no longer required.

**Pull request closed without merging**